docs/outlook/manifests.md

```diff
@@ -1,407 +1,66 @@
+# [XML Manifest](#tab/xmlmanifest)
 
-The following example uses the **DefaultMinVersion** attribute of the **\<Sets\>** element to require office.js version 1.1 or higher, and the **MinVersion** attribute of the **\<Set\>** element to require the Mailbox requirement set version 1.1.
+The **\<Permissions\>** element contains the required permissions for the add-in. In general, you should specify the minimum necessary permission that your add-in needs, depending on the exact methods that you plan to use. For example, a mail add-in that activates in compose forms and only reads but doesn't write to item properties like [item.requiredAttendees](/javascript/api/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.item#properties), and doesn't call [mailbox.makeEwsRequestAsync](/javascript/api/requirement-sets/outlook/preview-requirement-set/office.context.mailbox#methods) to access any Exchange Web Services operations should specify **ReadItem** permission. The following is an example of setting an Outlook permission.
 
 ```XML
 <OfficeApp>
 ...
-  <Requirements>
-    <Sets DefaultMinVersion="1.1">
-      <Set Name="MailBox" MinVersion="1.1" />
-    </Sets>
-  </Requirements>
+  <Permissions>ReadWriteItem</Permissions>
 ...
 </OfficeApp>
 ```
 
-## Form settings
-
-The **\<FormSettings\>** element is used by older Outlook clients, which only support schema 1.1 and not **\<VersionOverrides\>**. Using this element, developers define how the add-in will appear in such clients. There are two parts - **ItemRead** and **ItemEdit**. **ItemRead** is used to specify how the add-in appears when the user reads messages and appointments. **ItemEdit** describes how the add-in appears while the user is composing a reply, new message, new appointment or editing an appointment where they are the organizer.
-
-These settings are directly related to the activation rules in the **\<Rule\>** element. For example, if an add-in specifies that it should appear on a message in compose mode, an **ItemEdit** form must be specified.
-
-For more details, please refer to the [Schema reference for Office Add-ins manifests (v1.1)](../develop/add-in-manifests.md).
 
-## App domains
+# [Unified manifest for Microsoft 365 (developer preview)](#tab/jsonmanifest)
 
 The domain of the add-in start page that you specify in the **\<SourceLocation\>** element is the default domain for the add-in. Without using the **\<AppDomains\>** and **\<AppDomain\>** elements, if your add-in attempts to navigate to another domain, the browser or webview control will open a new window outside of the add-in pane. In order to allow the add-in to navigate to another domain within the add-in pane, add an **\<AppDomains\>** element and include each additional domain in its own **\<AppDomain\>** sub-element in the add-in manifest.
 
```

Comment:
```suggestion
```
Leftover from previous content?

Reply:
Strange. This paragraph isn't in the source file. I wonder why it's showing up here.